File: Beginner/1036 - Bhaskara's Formula/1036.c
#include <stdio.h>
#include <math.h>


int main(){

    double A, B, C;
    double x1, x2, delta;

    scanf("%lf %lf %lf", &A, &B, &C);

    delta = pow(B, 2) - (4*A*C);
    x1 = (-B + sqrt(delta))/(2*A);
    x2 = (-B - sqrt(delta))/(2*A);

    if(A>0 &&   delta>0){
        printf("R1 = %.5lf\n", x1);
        printf("R2 = %.5lf\n", x2);
    }
    else
        printf("Impossivel calcular\n");

    return 0;
}

File: Beginner/1036 - Bhaskara's Formula/1036.cs
using System;

class URI
{
    static void Main(string[] args)
    {
        double A, B, C;
        double delta, R1, R2;

        string[] input = Console.ReadLine().Split();
        A = double.Parse(input[0]);
        B = double.Parse(input[1]);
        C = double.Parse(input[2]);

        delta = B * B - 4 * A * C;

        if (delta < 0 || A == 0)
        {
            Console.WriteLine("Impossivel calcular");
        }
        else
        {
            R1 = (-B + Math.Sqrt(delta)) / (2 * A);
            R2 = (-B - Math.Sqrt(delta)) / (2 * A);
            Console.WriteLine($"R1 = {R1:F5}");
            Console.WriteLine($"R2 = {R2:F5}");
        }
    }
}